cog/cog-client.cpp

```diff
@@ -63,6 +66,51 @@ struct _CogClient {
 
 G_DEFINE_TYPE_WITH_PRIVATE (CogClient, cog_client, G_TYPE_OBJECT)
 
+enum {
+  PROP_REGION = 1,
+  N_PROPERTIES
+};
+
+static void
+cog_client_set_property (GObject *object,
+                         guint property_id,
+                         const GValue *value,
+                         GParamSpec *pspec)
+{
+  CogClient *self = COG_CLIENT (object);
+  CogClientPrivate *priv = GET_PRIVATE (self);
+
+  switch (property_id) {
+    case PROP_REGION:
+      priv->region = (CogRegion) g_value_get_enum (value);
+      break;
+    default:
+      /* We don't have any other property... */
+      G_OBJECT_WARN_INVALID_PROPERTY_ID (object, property_id, pspec);
+      break;
+  }
+}
+
+static void
+cog_client_get_property (GObject    *object,
```

Comment:
I wouldn't block the PR on this, but while you're removing the TODO commit... same style nitpick as in `set_property`.

Reply:
Thank you, I didn't see that.